### PR TITLE
Create CI builds

### PR DESCRIPTION
**Pull request by philipktlin**
Create CI builds config file for edgelet and libiothsm based on check-in builds.
- Update install.sh to clean app package list first before running apt-get update.

### CODE REVIEW

**Comment by myagley** [comment]
Can you please add a more descriptive PR title? The title is what will show up in the commit log.

**Comment by philipktlin** [comment]
> Can you please add a more descriptive PR title? The title is what will show up in the commit log.

Updated.  thanks.

**Comment by philipktlin** [comment]
Move clean up package command from install.sh to build config.

File: builds/ci/edgelet.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      - task: Bash@3
+        displayName: armv7-unknown-linux-gnueabihf build
+        inputs:
+          filePath: edgelet/build/linux/cross.sh
+          arguments: --toolchain armv7-unknown-linux-gnueabihf --release true
+      - task: Bash@3
+        displayName: armv7-unknown-linux-gnueabihf test
+        inputs:
+          filePath: edgelet/build/linux/cross-test.sh
+          arguments: --toolchain armv7-unknown-linux-gnueabihf --release true
```
[the remark this answers is not in the document]
Comment (myagley) [reply builds/ci/edgelet.yaml]: Is it possible to remove the `true` after the `--release`. `--release` is a flag, so it shouldn't need the explicit `true`.

[the remark this answers is not in the document]
Comment (philipktlin) [reply builds/ci/edgelet.yaml]: Current implementation requires "true" after --release; therefore leave it as-is.

File: builds/ci/edgelet.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        displayName: Install Rust
+        inputs:
+          filePath: edgelet/build/linux/install.sh
+      - script: cargo install --git https://github.com/myagley/cross.git --branch set-path
+        displayName: Install cross (fork with docker fix)
+      - task: Bash@3
+        displayName: armv7-unknown-linux-gnueabihf build
+        inputs:
+          filePath: edgelet/build/linux/cross.sh
+          arguments: --toolchain armv7-unknown-linux-gnueabihf --release true
```
[the remark this answers is not in the document]
Comment (philipktlin) [reply builds/ci/edgelet.yaml]: removed release option from check-in builds.

File: builds/ci/edgelet.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (myagley) [reply builds/ci/edgelet.yaml]: Ah I see. It's for our shell script. I get it now.

**Pull request merged**